src/client/styles/components/landing/footer.less

```diff
@@ -0,0 +1,5 @@
+.footer{
+    background-color: #F07C00;
```

Comment:
Also can we make the color to dark grey or black or navy blue? The orange might seem excessive given that the rest of the page will be orange

Reply:
Yeah sure, I'll post screenshots of some colors soon

Reply:
![Screenshot from 2019-12-26 15-58-19](https://user-images.githubusercontent.com/22998430/71490369-972e7b80-27f8-11ea-8ff5-728f412afb84.png)
@hanburger97 how's this?